apps/dsa.c

```diff
@@ -28,8 +28,8 @@ NON_EMPTY_TRANSLATION_UNIT
 typedef enum OPTION_choice {
     OPT_ERR = -1, OPT_EOF = 0, OPT_HELP,
     OPT_INFORM, OPT_OUTFORM, OPT_IN, OPT_OUT,
-    OPT_ENGINE, OPT_PVK_STRONG, OPT_PVK_WEAK,
-    OPT_PVK_NONE, OPT_NOOUT, OPT_TEXT, OPT_MODULUS, OPT_PUBIN,
+    OPT_ENGINE, OPT_PVK_NONE, OPT_PVK_WEAK, OPT_PVK_STRONG,
```

Comment:
Something like this : 

`    /* These OPT_PKV codes must stay ordered as it, see related case below */`